kong/plugins/key-auth/api.lua

```diff
@@ -26,7 +28,7 @@ return {
           return endpoints.handle_error(err_t)
         end
         if not consumer then
-          return responses.send_HTTP_NOT_FOUND()
+          return kong.response.exit(HTTP_NOT_FOUND, { Message = "Not found" })
```

Comment:
is the capital `Message` a mistake, shouldn't it be `message`?

Reply:
oops! thank you!